src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/GH1001.ProvideRequiredLocationProperties_Invalid.sarif

```diff
@@ -41,7 +41,7 @@
       "artifacts": [
         {
           "location": {
-            "uri": "FunctionalTestOutput.ValidateCommand/Inputs.GH1001.ProvideRequiredLocationProperties_Invalid.sarif",
+            "uri": "FunctionalTestOutput.ValidateCommand/GH1001.ProvideRequiredLocationProperties_Invalid.sarif",
```

Comment:
>GH1001 [](http://example.com/codeflow?start=57&length=6)

Here's the change mentioned earlier: we now elide 'Inputs.' from the manufactured test file name.

I have manually checked all the modified files in this directory to confirm that this is the sole change for each of them. (By all means feel free to double-check my work).

Note how much useful information is encoded in this URL about our test scenario, rather than being something like, 'foo.sarif', or 'mytest.sarif'.